core/random.rbs

```diff
@@ -129,141 +58,53 @@ class Random < Object
   #     [ rand, rand ]           # => [0.1915194503788923, 0.6221087710398319]
   #
   def self.srand: (?Integer number) -> Integer
-end
-
-# The default Pseudorandom number generator.  Used by class methods of Random.
-#
-#
-Random::DEFAULT: Random
```

Comment:
I ran `rbs prototype runtime --merge Random` and the `Random::DEFAULT` constant was removed from the definition file.

Do I need to specify any options when I run `rbs prototype`?